bin/src/cli.rs

```diff
@@ -446,7 +446,7 @@ pub enum TcpFrontendCmd {
             help = "frontend address, format: IP:port"
         )]
         address: SocketAddr,
-        #[clap(short = 't', long = "tag", help = "custom tags", parse(try_from_str = parse_tags))]
+        #[clap(long = "tag", help = "custom tags", parse(try_from_str = parse_tags))]
```

Comment:
I'd rather have `--tags` than `--tag`, with plural, and a more verbose help field, like

`help = "custom tags, provide them with 'key=value, other-key=other-value'"`